WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginEpilogueActivity.java

```diff
@@ -67,7 +70,12 @@ public void onConnectAnotherSite() {
 
     @Override
     public void onContinue() {
-        setResult(RESULT_OK);
+        if (mSiteStore.hasSite()) {
+            setResult(RESULT_OK);
+        } else if (AppPrefs.shouldShowPostSignupInterstitial()) {
```

Comment:
👍 I think that makes sense to store this in the app prefs and skip the interstitial during sign in. We have some users who regularly sign out and sign in back, and we might not want to show them the interstitial every time.

Note: See my next comment in the SignupEpiloguActivity.